src/coreclr/src/libraries-native/entrypoints.c

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __LIB_NATIVE_ENTRYPOINTS
+#define __LIB_NATIVE_ENTRYPOINTS
+#endif
+
+#ifdef FEATURE_GLOBALIZATION
+#include "pal_calendarData.h"
+#include "pal_casing.h"
+#include "pal_collation.h"
+#include "pal_locale.h"
+#include "pal_localeNumberData.h"
+#include "pal_localeStringData.h"
+#include "pal_icushim.h"
+#include "pal_idna.h"
+#include "pal_normalization.h"
+#include "pal_timeZoneInfo.h"
+#endif // FEATURE_GLOBALIZATION
+
+#ifndef BYTE
+typedef unsigned char BYTE;
+#endif
+
+#ifndef LPVOID
+typedef void *LPVOID;
+#endif
+
+#define FCFuncStart(name) extern const LPVOID name[]; const LPVOID name[] = {
+#define FCFuncEnd() (BYTE*)0x01 /* FCFuncFlag_EndOfArray */ };
+
+#define QCFuncElement(name,impl) \
+    (BYTE*)0x8 /* FCFuncFlag_QCall */, (LPVOID)(impl), (LPVOID)name,
+
+#if defined(__cplusplus)
```

Comment:
This should be always compiled as C. Do you really need this?

Reply:
I don't think so. Will remove.